story-generator/src/app/debug-panel/debug-panel.css

```
@@ -1,436 +1,136 @@
-    transform: translateY(0);
-  }
-  
-  .debug-toggle {
-    bottom: 80px;
-  }
+.debug-panel__error code {
+  display: block;
+  margin-top: 0.35rem;
+  font-size: 0.75rem;
+  color: #fecaca;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)